components/eventing-controller/pkg/backend/eventmesh/eventmesh.go

```diff
@@ -146,6 +146,13 @@ func (em *EventMesh) SyncSubscription(subscription *eventingv1alpha2.Subscriptio
 			log.Errorw("Failed to handle EventMesh subscription modified", errorLogKey, err)
 			return false, err
 		}
+
+		if featureflags.IsEventingWebhookAuthEnabled() && !isEventMeshSubModified {
```

Comment:
(nit) Can you add a comment explaining why we check for `!isEventMeshSubModified`?

Reply:
Done